pyaedt/modules/SolveSetup.py

```diff
@@ -1602,7 +1628,8 @@ def create_linear_count_sweep(
         freqstop : float
             Stopping frequency of the sweep.
         num_of_freq_points : int
-            Number of frequency points in the range.
+            Number of frequency points in the range. Default is 401 for
+            sweep types "Interpolating" or "Fast", and is 5 for "Discrete".
```

Comment:
```suggestion
            Number of frequency points in the range. The default is ``401`` for
            a sweep type of ``"Interpolating"`` or ``"Fast"``. The default is ``5`` for a sweep
            type of ``"Discrete"``.
```
I think I was confused by a similar entry earlier. The edits here are clearer and should be
used elsewhere.